packages/blockchain-api/src/currencyConversion/CurrencyConversionAPI.ts

```diff
@@ -74,14 +77,18 @@ export default class CurrencyConversionAPI<TContext = any> extends DataSource {
   private getSupportedExchangeRate(
     fromCode: string,
     toCode: string,
-    timestamp?: number
+    timestamp?: number,
+    knownExchangeRates?: { [pair: string]: string }
   ): Promise<BigNumber> {
     const pair = `${fromCode}/${toCode}`
 
     if (pair === 'cUSD/USD' || pair === 'USD/cUSD') {
       // TODO: use real rates once we have the data
       return Promise.resolve(new BigNumber(1))
     } else if (pair === 'cGLD/cUSD' || pair === 'cUSD/cGLD') {
+      if (knownExchangeRates && knownExchangeRates[pair]) {
```

Comment:
What is the expected behavior if it is a cGLD/cUSD or cUSD/cGLD transfer but the exchange rate is missing? To throw an error?

Reply:
We only want to use implied exchange rate when there was an exchange, otherwise we still ask the API.